nupic/research/frameworks/dendrites/modules/apply_dendrites.py

```diff
@@ -0,0 +1,64 @@
+# ----------------------------------------------------------------------
+# Numenta Platform for Intelligent Computing (NuPIC)
+# Copyright (C) 2020, Numenta, Inc.  Unless you have an agreement
```

Comment:
2021! Also, please update the license year in `functional/apply_dendrites.py` too, that file was also added this year.

Reply:
Got me again here. Fixed it.